core/vm/evmc.go

```diff
@@ -389,7 +389,9 @@ func (host *hostContext) Selfdestruct(evmcAddr evmc.Address, evmcBeneficiary evm
 		db.AddRefund(vars.SelfdestructRefundGas)
 	}
 	db.AddBalance(beneficiary, db.GetBalance(addr))
-	return db.SelfDestruct(addr)
+	db.SelfDestruct(addr)
+	// TODO (ziogaschr): check what to return
+	return true
```

Comment:
@meowsbits what do you think of this?

I was thinking to check if the address exists after the SelfDestruct and return it. Though we have to  further test on how this works.